index.js

```diff
@@ -26,7 +26,9 @@ export default async function npmUser(username) {
 		};
 	} catch (error) {
 		if (error.response.status === 404) {
-			error.message = 'User doesn\'t exist';
+			const notFoundError = new Error(`User \`${username}\` could not be found`, {cause: error});
+			notFoundError.code = 404;
```

Comment:
`.code` is generally a string in Node.js. For example: `'ERR_NO_USER'`.

Reply:
Updated.